src/csharp/Microsoft.Spark.Worker/Command/SqlCommandExecutor.cs

```diff
@@ -534,18 +535,23 @@ internal class ArrowGroupedMapCommandExecutor : SqlCommandExecutor
             ArrowStreamWriter writer = null;
             foreach (RecordBatch input in GetInputIterator(inputStream))
             {
-                RecordBatch result = worker.Func(input);
+                FxDataFrame dataFrame = new FxDataFrame(input);
```

Comment:
This looks like I'd expect. There should be zero-copying here, right?

I expect a similar change to the `ArrowSqlCommandExecutor` above.

Reply:
Yup, this is zero-copy.